### PR TITLE
Update test_1.py

### PR DESCRIPTION
**Pull request by sammarth-kapse**

### CODE REVIEW

**Review by dilipchauhan14 — APPROVED** [review]
Approving....

File: test_1.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Sammarth Kapse
```
Comment (sammarth-kapse) [inline test_1.py:1]: Added Author

File: test_1.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 from helper_functions import *
```
Comment (sammarth-kapse) [inline test_1.py:2]: Usual #1

File: test_1.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
```
Comment (sammarth-kapse) [inline test_1.py:3]: Usual #2

**Review by sammarth-kapse — COMMENTED** [review]
Usual ...

**Pull request merged**